Lib/urllib/parse.py

```diff
@@ -982,7 +1037,15 @@ def splitport(host):
             return host, port
     return host, None
 
-def splitnport(host, defport=-1):
+
+def splitnport(url=''):
```

Comment:
This was just careless.  Sorry that I made the mistake.  I've corrected all the signatures.